node-operators/rpc.md

```diff
@@ -0,0 +1,157 @@
+---
+
+# Run an RPC Node Using Docker
+
+## Introduction {: #introduction }
+
+Running an RPC node on a Tanssi ContainerChain allows you to connect to the network, discover other nodes to synch with through a bootnode, and offer an interaction layer for DApps over the WebSocket protocol.
+
+In this guide, you'll learn how to quickly spin up a ContainerChain node using [Docker](https://www.docker.com/){target=\_blank} on a Linux computer.
+
+!!! note
+    It is not possible to run an RPC node for Snap Appchains as they run on a private network, and their nodes are, therefore, unable to reach for synching.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running a Linux OS and install [Docker](https://docs.docker.com/desktop/install/linux-install/){target=\blank}.
+
+Run the following command to install Docker on a Linux Ubuntu platform:
+
+```bash
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+
+And the following command to check the installation:
+
+```bash
+sudo docker run hello-world
+```
+ 
+This is what a successful execution in the terminal looks like:
+
+--8<-- 'code/node-operators/rpc/terminal/hello-world.md'
+
+## Pulling the Docker Image {: #pulling-docker-image }
+
+As part of the automated deployment process, a Docker image is built for each [template](/learn/tanssi/included-templates/){target=\_blank} and published, bundling together the binary corresponding to the latest release of the [client node](/learn/framework/architecture/#architecture){target=\_blank} along with the [chain specification](/builders/build/customize/customizing-chain-specs/){target=\_blank} file.
+
+The chain specification is generated while registering the Appchain in the DApp, either using the provided parameters for the selected template or as uploaded when choosing the Custom Specs option.
+
+Execute the following command to pull the image that corresponds to the type of your ContainerChain:
+
+=== "EVM Template"
+
+    ```bash
+    docker pull moondancelabs/dancebox-container-chain-evm-templates
+    ```
+
+=== "Substrate"
+
+    ```bash
+    docker pull moondancelabs/dancebox-container-chain-simple-templates
+    ```
+
+The command will download and extract the image and show the status upon execution:
+
+--8<-- 'code/node-operators/rpc/terminal/pulling-docker-image.md'
+
+## Start-up Command {: #start-up-command }
+
+To spin up your node, you'll must run the Docker image with the `docker run` command.
+
+The following example spins up an RPC node with the entire history (blocks and state) of the chain for the [demo EVM ContainerChain](/builders/tanssi-network/networks/dancebox/demo-evm-containerchain/){target=\_blank} deployed on Dancebox with an id of 3001:
+
+```bash
+docker run -ti moondancelabs/dancebox-container-chain-evm-templates \
```

Comment:
You are adding a lot of unnecessary flags here. I would just show the most basic ones and offer some of the useful flags tbh

Reply:
Those flags are the ones ope are using when running the nodes, but yes, most of them are optional.
I'll leave only the required flags then!

Reply:
Yes I think that is best, and then just provide the most valuable flags etc.

Reply:
can you please remove the spacing in the front of each line?